api/cotrip_app/models.py

```diff
@@ -6,3 +6,17 @@ class Topic(models.Model):
 
     def __str__(self):
         return self.title
+
+class Location(models.Model):
```

Comment:
How does the Location model relate to the City model? Are they the same?

Reply:
I don't think there is a City model, so figured Location made more sense. That way for the city property we could put in something like "The Yucatan" or something like that.